ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.DesktopGL.CSharp/MGNamespace.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <MonoGameContentReference Include="**\*.mgcb" />
+    <MonoGameContentReference Include="Content\Content.mgcb" Visible="false" />
```

Comment:
Missing none include?

Reply:
Nope, everything gets autoincluded with either `Compile` or `None` in new csproj format.

Reply:
Ah, right!